spec/features/guests/add_room_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "Add a Room to the guest" do
+    it "should add the room to the guest and show it on the show page" do
+        hotel = Hotel.create!(name: "Caesar's")
+        room1 = hotel.rooms.create!(suite: "Presidential", rate: 400)
+        room2 = hotel.rooms.create!(suite: "Deluxe", rate: 200)
+        guest = Guest.create!(name: "John Lennon")
+        Booking.create!(guest: guest, room: room1)
+
+        visit "/guests/#{guest.id}"
+
+        expect(page).to have_content("John Lennon")
+        expect(page).to have_content("Presidential")
+        expect(page).to have_content("400")
+        expect(page).not_to have_content("Deluxe")
+        expect(page).not_to have_content("200")
```

Comment:
I like this testing of content NOT being there, then testing after we use the form, that it IS there. Nice job.